common/io/Serial.cpp

```diff
@@ -299,6 +295,8 @@ bool AcquireLockAndOpenSerialPort(const std::string &path, int oflag, int *fd) {
 void ReleaseSerialPortLock(const std::string &path) {
 #ifdef UUCP_LOCKING
   ReleaseUUCPLock(path);
+#else   // UUCP_LOCKING
+  OLA_INFO << "No unlock necessary for " << path;
```

Comment:
Can we change to one of these:
```suggestion
  OLA_INFO << "No unlock necessary with flock() for " << path;
```
or
```suggestion
  OLA_INFO << "No unlock necessary with for " << path << " as we're using flock()";
```